storage/couchdb/bootstrap.go

```diff
@@ -0,0 +1,106 @@
+package couchdb
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/flimzy/kivik"
+	_ "github.com/go-kivik/couchdb" //
+)
+
+func makeDB(client *kivik.Client, name string) error {
+	err := client.CreateDB(context.TODO(), name)
+	if err != nil {
+		exists, nerr := client.DBExists(context.TODO(), name)
```

Comment:
I added a test case now that tests the err != nil and returns nil as an error for a database that's being created again.

Reply:
Thank you!  Sorry for being so obtuse - I'm sure it's here, but I can't seem to spot this case.  Would you mind pointing it out to me, because I think I'm blind.  :)

Reply:
The test is here: https://github.com/theupdateframework/notary/blob/61c31cafe4211b8c67cc78fae3895e4538495470/server/storage/couch_realdb_test.go#L87